### DISCUSSION TITLE
pyarmor pack -e "--onefile " main.py but .exe file not working...

### DISCUSSION THREAD

**Discussion by developers-dev** [dashingsoft/pyarmor#1649] [Q&A]
pyarmor pack -e "--onefile " main.py
I created an exe file using the above command.
However, the exe file does not run.

Pyarmor Version 7.7.4
Pyinstaller Version 6.3.0
Python Version 3.9.12

What should I do? Please help.

**Comment by jondy** [comment]
Please first check Pyarmor 7.7.4 online documentation
